src/distill.c

```diff
@@ -31,7 +31,7 @@ void program_distill(void) {
   unsigned char status;
   int start = 0;
   float PI_value = 0.0;
-  int switch_ON = 0;
+  HeaterStatus switch_ON = HEATER_OFF;
   int initialcdn = 0;
   float roomtemperature = 0.0;
   int mode_change = 0;
```

Comment:
please use enums for `mode_change` in order to communicate what each mode does

Reply:
keine Ahnung was mode_change ist. Da musst Du Kotaro fragen. Ich hab distill.c nur geändert, weil ich den Rückgabewert von heater_switch() geändert habe.